test/decide_test.go

```diff
@@ -10,19 +10,28 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestImmediateDecide(t *testing.T) {
-	tsg := sim.NewTipSetGenerator(984615)
+func TestImmediateDecideAdversary(t *testing.T) {
+	immediateDecideAdversaryTest(t, 98562314)
+}
```

Comment:
Instead of doing this, call
```
f.Add(98562314)
```
in the Fuzz target, this will register `98562314` in the fuzz corpus and all corpus (both within `_test.go` and `testdata/fuzz`) is run during normal tests without timed fuzzing)

Reply:
nice thanks @Kubuxu

Reply:
Note: there's little point in doing this _plus_ checking in a seed corpus.

Reply:
Yes, in code f.Add is easier on the eyes than testdata corpus IMHO. Specially for testing fixed cases to keep a baseline coverage.